### DISCUSSION TITLE
伪代码如何使用其它格式

### DISCUSSION THREAD

**Discussion by xHan-233** [whutug/whu-thesis#269] [Q&A] [answered]
如何实现这样的伪代码格式？
![image](https://github.com/whutug/whu-thesis/assets/106260274/60584eb8-b171-4593-a20f-110dbb0a6f0c)

**Comment by SwitWu** [comment]
CTAN 上有关于伪代码排版的 Topic，见https://www.ctan.org/topic/pseudocode

**Reply by xHan-233** [reply]
好的，我去看看，感谢！

**Comment by Liuyerui** [comment]
请问改成功了吗，同问

**Reply by SwitWu** [reply, marked as answer]
直接在导言区 `\usepackage{algorithm2e}`，然后按其说明排版，有什么问题吗

**Reply by SwitWu** [reply]
见最新的 commit 1b70162ea93052b0b8f9ffff8dfa7cc85f39a209，有使用示例。

**Reply by Liuyerui** [reply]
好的谢谢
